### DISCUSSION TITLE
Math doesn't render in multi-paragraph markdown lists

### DISCUSSION THREAD

**Discussion by cdwilson** [community/community#18817] [Other Feature Feedback, Questions, & Ideas] [answered]
**Here's the source markdown:**

```
1. This works

    $$
    \sqrt{3x-1}+(1+x)^2
    $$
    
2. This also works

    Another paragraph

    $$
    \sqrt{3x-1}+(1+x)^2
    $$
    
3. This doesn't work $\sqrt{3x-1}+(1+x)^2$

    $$
    \sqrt{3x-1}+(1+x)^2
    $$

4. This also doesn't work $\sqrt{3x-1}+(1+x)^2$

    Another paragraph

    $$
    \sqrt{3x-1}+(1+x)^2
    $$

```

**Here's how it renders:**

1. This works

    $$
    \sqrt{3x-1}+(1+x)^2
    $$
    
2. This also works

    Another paragraph

    $$
    \sqrt{3x-1}+(1+x)^2
    $$
    
3. This doesn't work $\sqrt{3x-1}+(1+x)^2$

    $$
    \sqrt{3x-1}+(1+x)^2
    $$

4. This also doesn't work $\sqrt{3x-1}+(1+x)^2$

    Another paragraph

    $$
    \sqrt{3x-1}+(1+x)^2
    $$

**Comment by cdwilson** [comment, marked as answer]
Looks like this has been fixed 🎉 ...but there is still this issue https://github.com/orgs/community/discussions/28115

**Comment by nschloe** [comment]
This only worked for a short time. :frowning_face: The issue a dup of https://github.com/orgs/community/discussions/17325.
